src/app/user-page/user-page.component.scss

```diff
@@ -24,6 +24,10 @@ img[alt='Google avatar'] {
   background: $accent-3-dark;
 }
 
+.text-center-align {
```

Comment:
In general, if you need to create a new style, first check the global `styles.css` to see if something similar already exists.   If not, and your new style might be broadly useful, put it into the global `styles.css`, along with a comment that explains how it is/should be used.

Reply:
Can you please address this comment? Addressing it might be justifying via a comment why it's proper for it to be here, or actually moving the style, but I just want to make sure that you saw the comment and responded to it (maybe you already had, but there's no indication of it here in the PR).

Thanks!

Reply:
Yup I'm sorry I completely missed this comment! I did a simple keyword search in `styles.css`and found that there was no global style existing, I'll add it to the global file after double checking!